tests/rt/rt_tests.pm

```diff
@@ -0,0 +1,24 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Install and execute hackbench and cyclictest from rt-tests suite
```

Comment:
Well, I can see that :) Could you maybe state what is the *testing goal* of this test module and also what is the motivation? At best you also add a line below `# Maintainer` with `# Tags:` and reference testopia/poo/bsc/fate/jira with the requirements?